cpmonitor/static/js/drag-n-drop.js

```diff
@@ -42,6 +60,13 @@ $(document).ready(function () {
     target.classList.remove("drag-over-top", "drag-over-middle");
   }
 
+  function onDragLeaveColumnDropArea(ev) {
+    ev.preventDefault();
+    const columnDropAreaClass = ev.currentTarget.id;
+    const target = document.getElementById(columnDropAreaClass);
+    target.classList.remove("drag-over-middle");
+  }
+
```

Comment:
I'm not a frontend developer and maybe this is the problem but I don't really understand what this two methods are doing 🙈 Ok `onDragLeave` and `onDragOver` are the event types but what is meant by `area`? Could you may improve the naming? Maybe something like `onDragLeaveThenRemoveXY()`?

Reply:
Exactly onDragOver and onDragLeave define what happens when you move your mouse cursor over the html element that has these events attached or when your mouse cursor leaves that element. 
In this case they add the class "drag-over-middle" to the html element that styles the html element, in particular, it colors it grey so that the user sees that they can drop a task there. 
I named that area where a user can drop a task "columnDropArea" because it is an area (grey, rectangular patch) where a task can be dropped and it belongs to a column of the board. 
If you have a better name for that I can change it, but I am a bit uncreative :smile:. I named the functions `onDragOverColumnDropArea`because it specifies what happens when I drag over the "columnDropArea" because I did not want to go into detail about the styling in the function name. But you can make some suggestions! 

[Screencast from 05.04.2024 16:22:59.webm](https://github.com/GermanZero-de/localzero-monitoring/assets/14115305/5587b156-4f20-41eb-a663-20733188dea5)

Reply:
Ok know I understand your naming 😄  I thought "Drop" in `onDragOverColumnDropArea` is a verb and not a noun like "on drag over column then drop area". What do you think about `onDragOverDroppingArea` without column? Is the word column that important? If yes maybe `onDragOverDroppingAreaInColumn`?